yt/yt_proto/yt/client/misc/proto/workload.proto

```diff
@@ -10,6 +10,8 @@ message TWorkloadDescriptor
     required int32 band = 2;
     optional int64 instant = 3;
     repeated string annotations = 4;
+    optional string disk_fair_share_bucket_tag = 5;
```

Comment:
Do you think that the tag-weight pair is a fairly general description of a consumer? Perhaps it's worth expanding the interface and making at least a set of tags?

Reply:
> Do you think that the tag-weight pair is a fairly general description of a consumer?

Yes. We need to group load somehow and make fairness between groups of loads. Every request can be in one group only

> Perhaps it's worth expanding the interface and making at least a set of tags?

Not clear how will it work with set of tags and whether it will work expected at all
It this division should happen, a load should have a set of WorkloadDescriptors, not tags. Because different tags will have separate weights, categories, etc.